diambraArena/diambraEnvLib/diambra_pb2_grpc.py

```diff
@@ -84,123 +85,139 @@ def Close(self, request, context):
-            'EnvInit': grpc.unary_unary_rpc_method_handler(
-                    servicer.EnvInit,
-                    request_deserializer=diambra__pb2.EnvSettings.FromString,
-                    response_serializer=diambra__pb2.EnvInfoAndIntData.SerializeToString,
-            ),
-            'Reset': grpc.unary_unary_rpc_method_handler(
-                    servicer.Reset,
-                    request_deserializer=diambra__pb2.Empty.FromString,
-                    response_serializer=diambra__pb2.Obs.SerializeToString,
-            ),
-            'Step1P': grpc.unary_unary_rpc_method_handler(
-                    servicer.Step1P,
-                    request_deserializer=diambra__pb2.Command.FromString,
-                    response_serializer=diambra__pb2.Obs.SerializeToString,
-            ),
-            'Step2P': grpc.unary_unary_rpc_method_handler(
-                    servicer.Step2P,
-                    request_deserializer=diambra__pb2.Command.FromString,
-                    response_serializer=diambra__pb2.Obs.SerializeToString,
-            ),
-            'Close': grpc.unary_unary_rpc_method_handler(
-                    servicer.Close,
-                    request_deserializer=diambra__pb2.Empty.FromString,
-                    response_serializer=diambra__pb2.Empty.SerializeToString,
-            ),
+        'EnvInit': grpc.unary_unary_rpc_method_handler(
+            servicer.EnvInit,
+            request_deserializer=diambra__pb2.EnvSettings.FromString,
+            response_serializer=diambra__pb2.EnvInfoAndIntData.SerializeToString,
+        ),
+        'Reset': grpc.unary_unary_rpc_method_handler(
+            servicer.Reset,
+            request_deserializer=diambra__pb2.Empty.FromString,
+            response_serializer=diambra__pb2.Obs.SerializeToString,
+        ),
+        'Step1P': grpc.unary_unary_rpc_method_handler(
+            servicer.Step1P,
+            request_deserializer=diambra__pb2.Command.FromString,
+            response_serializer=diambra__pb2.Obs.SerializeToString,
+        ),
+        'Step2P': grpc.unary_unary_rpc_method_handler(
+            servicer.Step2P,
+            request_deserializer=diambra__pb2.Command.FromString,
+            response_serializer=diambra__pb2.Obs.SerializeToString,
+        ),
+        'Close': grpc.unary_unary_rpc_method_handler(
+            servicer.Close,
+            request_deserializer=diambra__pb2.Empty.FromString,
+            response_serializer=diambra__pb2.Empty.SerializeToString,
+        ),
     }
     generic_handler = grpc.method_handlers_generic_handler(
-            'diambra.EnvServer', rpc_method_handlers)
+        'diambra.EnvServer', rpc_method_handlers)
     server.add_generic_rpc_handlers((generic_handler,))
 
+# This class is part of an EXPERIMENTAL API.
+
 
- # This class is part of an EXPERIMENTAL API.
 class EnvServer(object):
     """The DIAMBRA service definition.
     """
 
     @staticmethod
     def EnvInit(request,
```

Comment:
PEP8 says:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
https://peps.python.org/pep-0008/#function-and-variable-names

Any idea why autopep8 didn't fix this?